x-pack/plugins/cases/public/components/custom_fields/text/get_eui_table_column.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import type { CaseUI } from '../../../containers/types';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getEuiTableColumn = ({
```

Comment:
Should we set a `width`?

Reply:
I set one for now but will still play more with this in the last PR to go over the overall look of the table.